src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacPropertyAttributes/DacPropertyAttributesAnalyzer.cs

```diff
@@ -67,28 +67,28 @@ private static Task AnalyzePropertyAsync(SymbolAnalysisContext symbolContext, PX
 			if (!validSpecialTypes)
 				return;
 
-			CheckForPXDBCalcedAndUnboundTypeAttributes(symbolContext, pxContext, fieldAttributesRegister, property);
+			CheckForPXDBCalcedAndUnboundTypeAttributes(symbolContext, pxContext, fieldAttributesRegister, property, attributesWithInfos);
 
 			await CheckForFieldTypeAttributesAsync(property, symbolContext, pxContext, attributesWithInfos)
 					.ConfigureAwait(false);
 		}
 
 		private static void CheckForPXDBCalcedAndUnboundTypeAttributes(SymbolAnalysisContext symbolContext, PXContext pxContext,
-			FieldTypeAttributesRegister fieldAttributesRegister, IPropertySymbol propertySymbol)
+			FieldTypeAttributesRegister fieldAttributesRegister, IPropertySymbol propertySymbol,
+			List<(AttributeData Attribute, List<FieldTypeAttributeInfo> Infos)> attributesWithInfos)
 		{
 			symbolContext.CancellationToken.ThrowIfCancellationRequested();
 
-			var attributes = propertySymbol.GetAttributes();
 			var hasUnboundTypeAttribute = false;
 			var hasPXDBCalcedAttribute = false;
 
-			foreach (var attr in attributes)
+			foreach (var (attribute, info) in attributesWithInfos)
```

Comment:
it should be `infos` instead of `info` because there are multiple DTOs

Reply:
ok